src/System.Collections.Specialized/tests/HybridDictionary/HybridDictionary.ValuesTests.cs

```diff
@@ -2,39 +2,44 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using Xunit;
+using System.Collections.Generic;
+using System.Collections.Tests;
+using System.Diagnostics;
 
 namespace System.Collections.Specialized.Tests
 {
-    public class HybridDictionaryValuesTests
+    public class HybridDictionaryValuesTests : ICollection_NonGeneric_Tests
     {
-        [Theory]
-        [InlineData(0, 0)]
-        [InlineData(0, 1)]
-        [InlineData(5, 0)]
-        [InlineData(5, 0)]
-        [InlineData(50, 0)]
-        [InlineData(50, 1)]
-        public void CopyTo(int count, int index)
```

Comment:
Were we *only* testing the CopyTo function for the Values of HybridDictionary? Ouch.